.env

```diff
@@ -1,3 +1,2 @@
-
 REACT_APP_JWT_SECRET="iloveyou"
 PORT=4000
```

Comment:
add .env file in gitignore. try not to upload this file on github.. read: https://stackoverflow.com/questions/30696930/how-to-hide-env-file-from-github